doc/dev/release.md

```diff
@@ -80,6 +82,8 @@ The following sections, often directly copied from our [changelog][doc-changelog
 ```Markdown
 [Version as title, ex. v1.2.3]
 
+**NOTE** See how to upgrade your project to this version by checking the [Version Upgrade Guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/migration/version-upgrade-guide.md)
+
```

Comment:
@joelanford f you folks accepted then, I am committed to go there and update all.

Reply:
Add it in the post-release actions.